akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/Behaviors.scala

```diff
@@ -157,9 +144,27 @@ object Behaviors {
    * need and in fact should not use (close over) mutable variables, but instead
    * return a potentially different behavior encapsulating any state changes.
    */
-  def immutable[T](onMessage: (ActorContext[T], T) ⇒ Behavior[T]): Immutable[T] =
+  def receive[T](onMessage: (ActorContext[T], T) ⇒ Behavior[T]): Immutable[T] =
```

Comment:
We also have `def receiveMessage(ctx: ActorContext[T], msg: T)` in `ExtensibleBehavior`.
Do you think that naming is inconsistent now, since it's the one with two parameters? Should we rename that to `receive`?

Reply:
Oh, great catch -- that looks like a receive then